nibabies/cli/run.py

```diff
@@ -84,6 +84,12 @@ def get_parser():
         default=False,
         help="Use low-quality tools for speed - TESTING ONLY",
     )
+    parser.add_argument(
+        "--age",
```

Comment:
shouldn't this be extracted from the BIDS structure?

Reply:
our test cases are currently not in BIDS structure 😄 

but down the line, yes if possible - `participants.tsv` is not mandatory.